src/app/App.js

```diff
@@ -18,7 +18,7 @@ export const App = () => {
       <Counter initialValue={5} />
       <SearchForm
         initialValue={""}
-        onSearch={handleSearch}
+        onChange={handleSearch}
```

Comment:
why do we need this change?

Reply:
could you elaborate on it, please?

Reply:
SearchForm

- Test that after typing to the input and pressing Enter key, the "onChange" prop is called with proper value

But maybe "onSearch" word is clearer. I'll change.

Reply:
done